src/screens/AddCustomer.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { Typography } from '@material-ui/core';
+import BaseHeader from '../components/BaseComponents/BaseHeader';
+import TextField from '../components/TextField';
+import Checkbox from '../components/Checkbox';
+import Button from '../components/Button';
+import { withStyles, createStyles, Theme } from '@material-ui/core/styles';
+import { Link, RouteComponentProps } from 'react-router-dom';
+
+const styles = (theme: Theme) =>
+  createStyles({
+    root: {
+      flexGrow: 1,
+    },
+    header: {
+      marginTop: '20px',
+      color: theme.palette.text.primary,
+    },
+    content: {
+      margin: '0 25px',
+      textAlign: 'left',
+      color: theme.palette.text.primary,
+    },
+    form: {
+      margin: '10px -5px',
+    },
+    label: {
+      marginTop: '-55px',
+      marginBottom: '40px',
+    },
+  });
+
+interface AddCustomerProps extends RouteComponentProps {
+  classes: { root: string; header: string; content: string; form: string; label: string; };
+  location: any;
+}
+
+function AddCustomer(props: AddCustomerProps) {
+  const { classes, match } = props;
+
+  return (
+    <div className={classes.root}>
+      <div className={classes.header}>
```

Comment:
This div can be replaced entirely with 
`<BaseHeader title="Add New Customer" leftIcon="backNav" />`

Same applies as in EditCustomer and AddMeterReading